201-api-management-create-with-internal-vnet-application-gateway/README.md

```diff
@@ -0,0 +1,16 @@
+# Create API Management in Internal VNet with App Gateway
```

Comment:
move this sample to the quickstarts/Microsoft.ApiManagement folder and remove "201-" from the foldername

Reply:
Done